blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,138 @@
+
+    ![unfiltered topology view](images/owner_screenshot.png)
+
+* Test bed: 9 worker and 3 master nodes, AWS m5.2xlarge machines
+* LokiStack size: 1x.small
+
+Results:
+
+  Below table shows 90th Percentile query times for each table:
+
+  | Time Range | Loki      | Prometheus
+  | :--------: | :-------: | :----------:
+  | Last 5m    | 984 ms    | 99 ms
+  | Last 1h    | 2410 ms   | 236 ms
+  | Last 6h    | > 10 s    | 474 ms
+
+As time range to fetch network flows gets wider, Loki queries tends to get slower or timing out, while Prometheus queries is able render the data within fraction of a second.
+
+### Resource utilization:
+In our tests conducted on 3 different test beds with varied workloads and network throughput, when Network Observability is configured without Loki, total savings of Memory usage could be in range 45-65% and CPU utilzation could be lower by 10-20%<sup>*</sup>. Not to mention you will not need to provision and plan for additional object storage in public clouds for Loki, overall reducing the cost and improving operational efficiency significantly.
+
+In our perf tests, [kube-burner](https://github.com/kube-burner/kube-burner) workloads were used to generate several objects and create heavy network traffic. We used sampling rate of 1 for all below tests. To further describe each test bed:
+
+1. Test bed 1: node-density-heavy workload ran against 25 nodes cluster.
+2. Test bed 2: ingress-perf workload ran against 65 nodes cluster.
+3. Test bed 3: cluster-density-v2 workload ran against 120 nodes cluster
+
+Below graphs shows total vCPU, memory and storage usage for a recommended Network Observability stack  - flowlogs-pipeline, eBPF-agent, Kafka, Prometheus and optionally Loki for production clusters.
+
+![Compare total vCPUs utilized with and without Loki](<blogs/lokiless_netobserv/images/vCPUs consumed by NetObserv stack.png/Total vCPUs consumed.png>)
+![Compare total RSS utilized with and without Loki](<blogs/lokiless_netobserv/images/Memory consumed by NetObserv stack.png>)
+
+Let's look at the amount of estimated storage you'd need for all the network flows and Prometheus metrics that Network Observability has to store. For context, even when Loki is installed Network Observability publishes default set of Prometheus metrics for monitoring dashboards, and it adds additional metrics when Loki is disabled to visualize network flows. The graphs below shows the estimated amount of storage required to store 15 days of Network flows (when configured with Loki), Prometheus metrics and Kafka as intermediary data streaming layer between eBPF-agent and flowlogs-pipeline. 
```

Comment:
```suggestion
Let's look at the amount of estimated storage you need for all the network flows and Prometheus metrics that Network Observability has to store. For context, even when Loki is installed, Network Observability publishes a default set of Prometheus metrics for monitoring dashboards, and it adds additional metrics when Loki is disabled to visualize network flows. The graphs below show the estimated amount of storage required to store 15 days of network flows data (when configured with Loki), with Prometheus metrics and Kafka as an intermediary data streaming layer between the eBPF-agent and the flowlogs-pipeline. 
```

Reply:
Just verb tense and adding some the's and a's. One question, "network flows" only or "network flows data"?

Reply:
> "network flows" only or "network flows data"?
 
either should be fine.  Although I am not sure about "with Prometheus" here, wdyt?

Reply:
I think the "with" or alternatively "in addition to" is needed to introduce it because its a non-restrictive clause that is independent of the main sentence, but still a fragment.